internal/pkg/term/selector/selector.go

```diff
@@ -176,10 +181,17 @@ type WorkspaceSelect struct {
 	appName string
 }
 
-// PipelineSelect is a workspace pipeline selector.
-type PipelineSelect struct {
+// WsPipelineSelect is a workspace pipeline selector.
+type WsPipelineSelect struct {
 	prompt Prompter
-	ws     WorkspacePipelinesLister
+	ws     WsPipelinesLister
+}
+
+// CodePipelineSelect is a selector for deployed pipelines.
+type CodePipelineSelect struct {
+	*Select
```

Comment:
Do we need to embed `*Select`? it doesn't look like we're using it

Reply:
We use it in `askAppName()` for `o.sel.Application(...)`: https://github.com/aws/copilot-cli/pull/3382/files#diff-f7c23022b6f1c5454bea7d03c7570bbddf15b0f27bb1aa09d41421d0176bc5ceR180